### PR TITLE
formalize advisors

### PR DESCRIPTION
**Pull request by tpolecat**
This formalizes the Advisor role.

### CODE REVIEW

File: ADVISORS.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Advisors
+
+This document lists the Organization's advisors, who may be asked to lend guidance and assist in decision-making. Advisors do not vote and are not Steering Committee members.
+
+
+| **Name**           | **GutHub**                                 |
```
Comment (ChristopherDavenport) [inline ADVISORS.md:6]:
```suggestion
| **Name**           | **GitHub**                                 |
```

**Review by rossabaker — COMMENTED** [review]
:+1: on the charter change.

:+1: on both advisors, but I want to hear from them that they want to be listed as such.

**Review by SethTisue — APPROVED** [review]

**Review by ChristopherDavenport — APPROVED** [review]
Approved on both fronts.

**Review by mpilquist — APPROVED** [review]

**Review by rossabaker — APPROVED** [review]
I've heard from both named advisors.

**Review by djspiewak — APPROVED** [review]

**Comment by rossabaker** [comment]
Please don't construe this as me wanting to be chair, but since it's vacant (#24):
* a quorum is 5 votes
* the amendment is affirmed with at least 2/3 of voters, maximally 6
* it would be neat if a workflow did this math

**Comment by tpolecat** [comment]
Make that 5 since I can't approve my own PR, but :+1:

**Pull request merged**